src/wallet/wallet.cpp

```diff
@@ -2330,9 +2330,9 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
                         if (recipient.fSubtractFeeFromAmount && nFeeRet > 0)
                         {
                             if (txout.nValue < 0)
-                                strFailReason = _("The transaction amount is too small to pay the fee");
+                                strFailReason = _("The transaction output amount is too small to pay the fee");
                             else
-                                strFailReason = _("The transaction amount is too small to send after the fee has been deducted");
+                                strFailReason = _("The transaction output amount is too small to send after the fee has been deducted");
                         }
                         else
                             strFailReason = _("Transaction amount too small");
```

Comment:
Do you want to include this line too?
Not sure it's worth changing these, but if you are going to, perhaps you could be even more clear
"One of the transaction output amounts .." or similar